src/assets/custom/css/_sass/_single-training.scss

```diff
@@ -1,9 +1,30 @@
-.single-training__instructor-bio {
-  @include small-and-medium {
+.single-training__accordion {
+  @include small {
+    padding-bottom: 350px;
```

Comment:
This looks like it may be used to create an overlap? If so we should use the `$veretical-overlap` var